gallery/gallery/README.md

```diff
@@ -0,0 +1,7 @@
+# gallery
```

Comment:
If the README in /gallery is meant to be the one for the whole gallery project, I'd remove this one.

Reply:
You're right, removed